src/components/TokenBalanceCard/TokenBalanceCardTemplate.tsx

```diff
@@ -37,25 +46,26 @@ const TokenBalanceCardTemplate: FC<Props> = ({
   ...restProps
 }) => {
   return (
-    <Card {...restProps} as="section">
-      <HStack as="header" mb="4">
-        <Icon boxSize="16px" as={icon} />
-        <LabelSm>{title}</LabelSm>
+    <VStack spacing={1} alignItems="flex-start" {...restProps}>
+      <H2 fontSize={16} lineHeight="24px" color="#808080" fontWeight="medium">
+        {title}
+      </H2>
+      <HStack alignItems="baseline" fontSize={24} lineHeight="40px">
+        <TokenBalance
+          tokenAmount={tokenBalance}
+          usdBalance={usdBalance}
+          tokenDecimals={tokenDecimals}
+          tokenFormat={tokenFormat}
+          withHigherPrecision={withHigherPrecision}
+          precision={precision}
+          higherPrecision={higherPrecision}
+          isEstimated
+          fontWeight="black"
+          color="white"
```

Comment:
This is the only place where we use `TokenBalance` component (but please double-check this), so we could remove all the properties that are not used anymore (like `isLarge for example).

We could also move the `tokenSymbol` text below inside `TokenBalance` and replace the old one here, but I'll leave that decision to you. I think it would make sense to keep it closely tied to the `TokenBalance` component next to it's amount. If you're not gonna replace it then we can also remove `withSymbol` and `tokenSymbol` properties from `TokenBalance`. Please check for other unused properties and remove them 😉

Reply:
<img width="265" alt="image" src="https://github.com/threshold-network/bitcoin-on-base/assets/11503879/5590fc83-f427-4d84-b6eb-06df4d266ac7">

There is actually large variant of the component I forgot to account. I introduced it with this commit: 3623cd1b

Also I've removed redundant props like boolean flags that can be defined directly by the presence of the value.
Example: to display values with higher precision you have to define both `higherPrecision` prop which is pretty much self explanatory and `withHigherPrecision` prop - which is counter intuitive. If I define higher precision, I'd like to display such value, that said the boolean `withHigherPrecision` is redundant.